### PR TITLE
Version 4.0, add license, update recipe

### PR DESCRIPTION
**Pull request by dbast**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by dbast** [comment]
@conda-forge/core no maintainer activities here since long. Ready for review/merge :)

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,60 +1,51 @@
...
     - pyrsa-verify --help
-    - pyrsa-encrypt-bigfile --help
-    - pyrsa-decrypt-bigfile --help
 
 about:
   home: https://stuvel.eu/rsa
   license: Apache 2.0
   summary: 'Pure-Python RSA implementation'
+  license_family: APACHE
+  license_file: LICENSE
```
Comment (scopatz) [inline recipe/meta.yaml:47]: should these go next to license?

**Comment by scopatz** [comment]
Are you interested in adding yourself as a maintainer?

**Comment by dbast** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml (hunk 1 above)
Comment (dbast) [reply recipe/meta.yaml:47]: Makes sense, done. Also added myself as maintainer.

**Comment by dbast** [comment]
@conda-forge-admin, please rerender

**Comment by dbast** [comment]
@scopatz Ready to merge :)

**Comment by scopatz** [comment]
thanks!

**Pull request merged**